src/main/java/edu/cornell/kfs/concur/service/ConcurReportsService.java

```diff
@@ -2,11 +2,16 @@
 
 import edu.cornell.kfs.concur.businessobjects.ConcurReport;
 import edu.cornell.kfs.concur.businessobjects.ValidationResult;
+import edu.cornell.kfs.concur.rest.xmlObjects.ConcurEventNotificationListDTO;
 
 public interface ConcurReportsService {
     
     ConcurReport extractConcurReport(String reportURI);
 
     void updateExpenseReportStatusInConcur(String workflowURI, ValidationResult validationResult);
+    
+    boolean deleteFailedEventQueueItem(String noticationId);
+    
+    ConcurEventNotificationListDTO retrieveConcurEventNotificationListDTO();
```

Comment:
It might be a good idea to rename this method, to provide a clearer description of what data it is returning.